### DISCUSSION TITLE
App icon

### DISCUSSION THREAD

**Discussion by saf-dmitry** [notaapp/talk#194] [Polls]
I like the old macOS app icon much more than the new one. To me the new icon looks a bit dated (probably, too skeuomorphic) and
totally inexpressive. The old one was simple, clear and distinctive.

Old (v0.34):

<img width="512" alt="Nota" src="https://user-images.githubusercontent.com/30050534/164061836-95d56a2f-c376-445d-b05b-c0b1137d7718.png">

New (v0.35):

<img width="512" alt="icon" src="https://user-images.githubusercontent.com/30050534/164061872-c799f305-b790-4dba-bbce-1b59059ad3f2.png">

Which icon would you prefer?

**Comment by davistech** [comment]
I like the old one.

**Comment by erusev** [comment]
Thanks to everyone for the input! We'll try to improve the icon for the next release.

**Comment by erusev** [comment]
Here are a few more variations:

https://www.figma.com/file/VFbnYxb7ygUltpcTzU7l4b/

Most are around the letter "n", and the last one around the idea of a "note", which is what the word Nota means, and the cube-like shape is about notes as building blocks of a knowledge base, which is the primary use case of the app.

I also included some of our influences (on the right).

Any feedback would be super helpful 🙏

**Reply by webwesen** [reply]
1st row, white background gets my vote

**Comment by vladstudio** [comment]
Hey! I got a bit inspired and here is my version - it is quite similar to the current one, but I applied just a tiny bit of effects, and I also found an excellent "n", geometric and with a fun little tail -

https://www.figma.com/file/yqoiPt3jzq2kOluqJKJ86U/Nota?node-id=2%3A2

[icon.icns.zip](https://github.com/notaapp/talk/files/9159371/icon.icns.zip)

![icon](https://user-images.githubusercontent.com/914320/180215217-8666431a-42ae-4005-ae1e-ac52acc1ccc8.png)

**Reply by astoilkov** [reply]
Ooo. 😍. @erusev What do you think? I like the effects a lot.

**Reply by vladstudio** [reply]
thanks! if you're interested, the font is https://fonts.google.com/specimen/Righteous?preview.text=nota&preview.text_type=custom - awful for long texts, but great for short logos :-)

**Reply by Arbiter313** [reply]
It's nice but I have a soft spot for the current logo 👀

**Reply by erusev** [reply]
Looks great 😊 Thanks for sharing 🙏
